Test/AccountTest.cs

```diff
@@ -29,7 +29,8 @@ public void CreateAccountWithParameters()
                 VatNumber = "my-vat-number",
                 TaxExempt = true,
                 EntityUseCode = "I",
-                CcEmails = "cc1@test.com,cc2@test.com"
+                CcEmails = "cc1@test.com,cc2@test.com",
+                Address = new Address()
```

Comment:
We're actually going to need a public constructor for `Address`. The only reason this works in the test is because the test are `internal` to the library.